packages/coreutils/src/url.ts

```diff
@@ -26,6 +26,9 @@ export namespace URLExt {
     return urlparse(url);
   }
 
+  export function getHostName(url: string): string {
```

Comment:
Can you please add a docstring for this function?

Reply:
Done. Thanks for your comment.